### ISSUE TITLE
False positives on multi-line functions indentation starting with v3.2

### ISSUE THREAD

**Issue by acelaya** [squizlabs/PHP_CodeSniffer#1824]
Hi!

I have started to get false positives with the mesage "Multi-line function call not indented correctly; expected 56 spaces but found 58"

The block marking the error is this one

```php
$req = ServerRequestFactory::fromGlobals()->withAttribute('identity', $user)
                                          ->withAttribute(
                                              'transactionId',
                                              (string) $transaction->getPublicId()
                                          );
```

It works if a remove 2 spaces from the second `withAttribute` call, like this

```php
$req = ServerRequestFactory::fromGlobals()->withAttribute('identity', $user)
                                        ->withAttribute(
                                            'transactionId',
                                            (string) $transaction->getPublicId()
                                        );
```

However, this didn't use to mark an error, and now it does. I like to keep methods aligned when using method chaining.

I'm just using the PSR-2 rule.

**Comment by jrfnl** [comment]
Duplicate of #1793 ? This has already been fixed in `master`. The fix will be in the 3.2.3 release.

**Comment by acelaya** [comment]
Great! I'll try master branch and let you know


--
Alejandro Celaya Alastrué

**Comment by acelaya** [comment]
Can confirm that using `dev-master` everything works again as in ^3.1 in my project :-)

**Comment by gsherwood** [comment]
Thanks for testing that fix.
